%% start of file `template.tex'.
%% Copyright 2006-2013 Xavier Danaux (xdanaux@gmail.com).
%
% This work may be distributed and/or modified under the
% conditions of the LaTeX Project Public License version 1.3c,
% available at http://www.latex-project.org/lppl/.


\documentclass[11pt,letterpaper,sans]{moderncv}        % possible options include font size ('10pt', '11pt' and '12pt'), paper size ('a4paper', 'letterpaper', 'a5paper', 'legalpaper', 'executivepaper' and 'landscape') and font family ('sans' and 'roman')

% modern themes
\moderncvstyle{banking}                            % style options are 'casual' (default), 'classic', 'oldstyle' and 'banking'
\moderncvcolor{blue}                                % color options 'blue' (default), 'orange', 'green', 'red', 'purple', 'grey' and 'black'
\renewcommand{\familydefault}{\sfdefault}         % to set the default font; use '\sfdefault' for the default sans serif font, '\rmdefault' for the default roman one, or any tex font name
\nopagenumbers{}                                  % uncomment to suppress automatic page numbering for CVs longer than one page

% character encoding
\usepackage[utf8]{inputenc}
\usepackage{fontawesome}
\usepackage{fontspec}
\usepackage{tabularx}
\usepackage{ragged2e}
\usepackage{tabto}
% if you are not using xelatex ou lualatex, replace by the encoding you are using
%\usepackage{CJKutf8}                              % if you need to use CJK to typeset your resume in Chinese, Japanese or Korean

% adjust the page margins
\usepackage[margin=0.5in]{geometry}
\usepackage{multicol}
%\setlength{\hintscolumnwidth}{3cm}                % if you want to change the width of the column with the dates
%\setlength{\makecvtitlenamewidth}{7cm}           % for the 'classic' style, if you want to force the width allocated to your name and avoid line breaks. be careful though, the length is normally calculated to avoid any overlap with your personal info; use this at your own typographical risks...

\usepackage{import}

% personal data
\name{Karthik}{Ram Padmanabhan, Ph.D.}
% \title{Curriculum Vitae}                               % optional, remove / comment the line if not wanted
\address{San Francisco, CA 94131 }{}{}% optional, remove / comment the line if not wanted; the "postcode city" and and "country" arguments can be omitted or provided empty
% \phone[mobile]{909-839-3097}                   % optional, remove / comment the line if not wanted
% \phone[fixed]{01234 123456}                    % optional, remove / comment the line if not wanted
%\phone[fax]{+3~(456)~789~012}                      % optional, remove / comment the line if not wanted
% \email{xpan1@swarthmore.edu}                               % optional, remove / comment the line if not wanted
% \homepage{shawnpan.me}                         % optional, remove / comment the line if not wanted
% \extrainfo{}                 % optional, remove / comment the line if not wanted
%\photo[64pt][0.4pt]{picture}                       % optional, remove / comment the line if not wanted; '64pt' is the height the picture must be resized to, 0.4pt is the thickness of the frame around it (put it to 0pt for no frame) and 'picture' is the name of the picture file
%\quote{Some quote}                                 % optional, remove / comment the line if not wanted

% to show numerical labels in the bibliography (default is to show no labels); only useful if you make citations in your resume
%\makeatletter
%\renewcommand*{\bibliographyitemlabel}{\@biblabel{\arabic{enumiv}}}
%\makeatother
%\renewcommand*{\bibliographyitemlabel}{[\arabic{enumiv}]}% CONSIDER REPLACING THE ABOVE BY THIS

% bibliography with mutiple entries
%\usepackage{multibib}
%\newcites{book,misc}{{Books},{Others}}

  
\newcommand*{\customcventry}[7][.25em]{
  \begin{tabular}{@{}l} 
    {\bfseries #4}
  \end{tabular}
  \hfill% move it to the right
  \begin{tabular}{l@{}}
     {\bfseries #5}
  \end{tabular} \\
  \begin{tabular}{@{}l} 
    {\itshape #3}
  \end{tabular}
  \hfill% move it to the right
  \begin{tabular}{l@{}}
     {\itshape #2}
  \end{tabular}
  \ifx&#7&%
  \else{\\%
    \begin{minipage}{\maincolumnwidth}%
      \small#7%
    \end{minipage}}\fi%
  \par\addvspace{#1}}

\newcommand*{\customcvproject}[4][.25em]{
%   \vfill\noindent
  \begin{tabular}{@{}l} 
    {\bfseries #2}
  \end{tabular}
  \hfill% move it to the right
  \begin{tabular}{l@{}}
     {\itshape #3}
  \end{tabular}
  \ifx&#4&%
  \else{\\%
    \begin{minipage}{\maincolumnwidth}%
      \small#4%
    \end{minipage}}\fi%
  \par\addvspace{#1}}

\setlength{\tabcolsep}{12pt}

%----------------------------------------------------------------------------------
%            content
%----------------------------------------------------------------------------------
\begin{document}
%\begin{CJK*}{UTF8}{gbsn}                          % to typeset your resume in Chinese using CJK
%-----       resume       ---------------------------------------------------------
\makecvtitle
\vspace*{-15mm}

\begin{center}
\begin{tabular}{ c c c c }
 \faEnvelopeO\enspace \href{mailto:karthikrpad@gmail.com}{karthikrpad@gmail.com} & \faMobile\enspace 812-325-0180 & \faGlobe\enspace \href{https://www.karthikrp.com}{karthikrp.com} & \faLinkedin\enspace\href{https://www.linkedin.com/in/karthikrpad}{karthikrpad}\\  
\end{tabular}
\end{center}

\section{PROFESSIONAL SUMMARY}
{-  Bioinformatics Analyst with 10+ years of hands-on experience in development of next-gen sequencing pipelines \\
- Expert in analysis of high-throughput sequencing data, highly collaborative with excellent communication skills}

\section{TECHNICAL SKILLS}

% {\customcvproject{Software/Tools}{}
% {- Unix OS, R Bioconductor, RStudio, AWS, bioinformatics tools used in downstream analysis (QC, trimming, alignment, peak-calling, variant calling, sequence annotation, etc.), pathway/regulatory network analysis (iPathwayGuide), visualization (Cytoscape, R Shiny), Machine learning and feature selection (Random forests, Genetic Algorithms, Support Vector Machines), \LaTeX
%   }
% }
{\customcvproject{Biological Data Analysis}{}
{- bulk and scDNA-Seq (CNV and SNV calling, Bisulfite Sequencing, ChIP-Seq, (h)MeDIP-Seq, ATAC-Seq)\\
- bulk and scRNA-Seq (Expression profiling, de novo transcriptome assembly)}
}

{\customcvproject{Languages/Software Development}{}
{- R, Python, Perl, Nextflow, AWS, R Shiny, Snakemake, Docker, Singularity, JavaScript, version control (CVS, git), Jupyter Notebook, SDLC best practices\\
- Machine learning and feature selection (Random forests, Genetic Algorithms, Support Vector Machines), \LaTeX}
}

% {\customcvproject{Databases}{}
% {- UCSC Genome Browser, NCBI, KEGG, Reactome, GEO, SRA, ENCODE, Gene Ontology, TCGA}
% }

\section{BIOINFORMATICS EXPERIENCE}

{\customcventry{San Jose, CA}{Takara Bio USA}{Staff Scientist, Bioinformatics}{Jan 2021- present}{}
{- Led the development and maintenance of Cogent suite of data analysis pipelines for DNA/RNA-Seq data using Nextflow\\ 
- Worked on algorithm and UI development for Embgenix PGT-A software for aneuploidy detection (low pass sequencing CNV detection) in embryos\\ 
- Collaborated with bench scientists to address bioinformatics needs in DNA-Seq, particularly in reproductive health
%- Developed and maintained pipelines and software to aid design of new products in R\&D using Python, Shiny and JavaScript
}

{\customcventry{Ann Arbor, MI}{University of Michigan}{Bioinformatician/Computational Biologist}{Sep 2016 - Jan 2021}{}
{- Developed, tested and maintained bioinformatics pipelines using Snakemake, R, Python, and open-source bioinformatics tools\\
  - Worked with scientists to develop analysis plans and budgets for bioinformatics/epigenomics research groups\\
  - Applied computational and statistical tools to interpret results from high dimensional genomics experiments derived from high-throughput sequencing (ChIP-Seq, ERRBS/WGBS, bulk/single-cell ATAC-Seq, RNA-Seq, Exome-Seq)\\
  - Organized pre- and post-analysis consultations, generated detailed analysis reports and interpreted results effectively to non-expert audience
}
}

{\customcventry{West Lafayette, IN}{Purdue University}{Research Assistant}{Aug 2012 - Jul 2016}{}
{- Analyzed NGS data (\textit{de novo} transciptome analysis) from giant ragweed to identify genes responsible for disease resistance\\
  - Performed domain co-occurrence analysis using statistical models for identifying the pairwise probability of protein domains}
}

{\customcventry{Chesterfield, MO}{Monsanto}{Data Science Intern}{May 2015 - Aug 2015}{}
{- Investigated the effect of metabolites and genes on selected yield-related phenotypes in corn using machine learning algorithms and feature selection methods, generated a list of predictive features affecting the yield of corn\\
  - Collaborated with multi-state teams to design and develop web applications in R Shiny, communicated results effectively with non-specialist scientists from other teams}
}

\section{EDUCATION}
{\customcventry{West Lafayette, IN}{Purdue University}{Ph.D. Biological Sciences (Computational Life Sciences Program)}{Jul 2016}{}{}}
{\customcventry{Bloomington, IN}{Indiana University}{M.S. Bioinformatics}{Jun 2011}{}{}}
{\customcventry{Tamil Nadu, India}{SASTRA University}{B.Tech. Industrial Biotechnology}{May 2009}{}{}}

\section{PUBLICATIONS AND PATENTS}
\begin{minipage}{\maincolumnwidth}%
	\small{
	- J. Meyers, L. Zhao, N. Yasuyama, J. Kim, \textbf{K. Padmanabhan}, et al. "Evaluation of Cell-Free Nucleic Acids in Blastocoel Fluid Conditioned Media and its Potential for Non-Invasive Embryo Assessment". \textit{Fertility and Sterility}. 2022 Oct. doi: 10.1016/j.fertnstert.2022.09.179\\
	- N.K. Patel, J.H. Nunez, M. Sorkin, S. Marini, C.A. Pagani, A.L. Strong, C.D. Hwang, S. Li, \textbf{K. Padmanabhan}, et al. "Macrophage TGF-$\beta$ signaling is critical for wound healing with heterotopic ossification after trauma".  \textit{JCI Insight}. 2022 Sep. doi: 10.1172/jci.insight.144925.\\
	- J. Foox, J. Nordlund, C. Lalancette, T. Gong, M. Lacey, S. Lent, B.W. Langhorst, V.K.C. Ponnaluri, L. Williams, \textbf{K. Padmanabhan}, et al. "The SEQC2 Epigenomics Quality Control (EpiQC) Study: Comprehensive Characterization of Epigenetic Methods, Reproducibility, and Quantification".  \textit{Genome Biology}. 2021 Dec. doi: 10.1186/s13059-021-02573-y.\\
	- J. Meyers, J. Laliberte, N. Yasuyama, J. Kim, \textbf{K. Padmanabhan}, et al. "The Effect of Embryo Morphology and Culturing Methods on the Performance of Non-Invasive Preimplantation Genetic Testing for Aneuploidies". \textit{Fertility and Sterility}. 2021 Sep. doi: 10.1016/j.fertnstert.2021.07.1053 \\
	- Z.M. Laubach, J.R. Greenberg, J.W. Turner, T. Montgomery, M.O. Pioon, L. Smale, R. Cavalcante, \textbf{K. Padmanabhan}, et al. "Early-life social experience affects offspring DNA methylation and later life stress phenotype". \textit{Nature Communications}. 2021 Jul. doi:10.1038/s41467-021-24583-x.\\
	- N.J. Edwards, E. Hobson, D. Dey, A. Rhodes, C. Pagani, A.L. Strong MD, G.E. Hespe, A. Huber, \textbf{K. Padmanabhan}, et al. "High Frequency Spectral Ultrasound Imaging Detects Early Stage Post Traumatic Heterotopic Ossification in Rodent Models". \textit{Stem Cells Dev}. 2021 May. doi: 10.1089/scd.2021.0011.\\
	- C.A. Pagani, A.K. Huber, C. Hwang, S. Marini, \textbf{K. Padmanabhan}, et al. "Novel Lineage Tracing System to Identify Site-specific Ectopic Bone Progenitor Cells". \textit{Stem Cell Reports}. 2021 Feb. doi: 10.1016/j.stemcr.2021.01.011.\\
          - P. Venkatraman, I. Mills-Henry, \textbf{K. Padmanabhan}, et al. “Rods Contribute to Visual Behavior in Larval Zebrafish”. \textit{Investigative Ophthalmology and Visual Science}. 2020 Oct. doi: 10.1167/iovs.61.12.11\\
	- A.K. Huber, N. Patel, C.A. Pagani, S. Marini, \textbf{K. Padmanabhan}, et al. "Immobilization after injury alters extracellular matrix and stem cell fate". \textit{The Journal of Clinical Investigation}. 2020 Jul. doi: 10.1172/jci136142.\\
          - A.K Saha, R. Contreras-Galindo, Y.S. Niknafs, M. Iyer, T. Qin, \textbf{K. Padmanabhan}, et al. "The role of the histone H3 variant CENPA in prostate cancer". \textit{Journal of Biological Chemistry}. 2020 May. doi: 10.1074/jbc.RA119.010080\\
	- \textbf{K. Padmanabhan}, K. Segobye, S.C. Weller, B. Schulz, M. Gribskov. "Preliminary investigation of glyphosate resistance mechanism in giant ragweed using transcriptome analysis". \textit{F1000 Research}. 2016 Jun. doi: 10.12688/f1000research.8932.1\\
	- B. Schulz, S.C. Weller, M. Gribskov, \textbf{K. Padmanabhan}, K. Segobye, "Diagnostic Tools for Herbicide Resistance in Weeds", Application No. 61/910,770 (“Technology”), 2013
	}%
\end{minipage}%



\section{PRESENTATIONS}
\begin{minipage}{\maincolumnwidth}%
	\small{
	- \textbf{K. Padmanabhan}, K. Segobye, S.C. Weller, B. Schulz, M. Gribskov, "Transcriptome Analysis of giant ragweed", Office of Interdisciplinary Graduate Programs Spring Reception, Purdue University, 2015 (Poster Presentation)\\
	- \textbf{K. Padmanabhan}, K. Segobye, M. Gribskov, B. Schulz, S.C. Weller, "Molecular Analysis of Glyphosate Resistance in Giant Ragweed", NCWSS Annual Meeting, Minneapolis, MN, December 3, 2014 (Oral Presentation)\\
	- \textbf{K. Padmanabhan}, K. Segobye, M. Gribskov, B. Schulz, S.C. Weller, "Molecular Analysis of Glyphosate Resistance in Giant Ragweed", Graduate Student Retreat, Purdue University, 2014 (Oral Presentation)\\
          - \textbf{K. Padmanabhan}, N.B. Best, M. Gribskov, S.C. Weller, B. Schulz, "Transcriptome Analysis of Glyphosate Resistance in Giant Ragweed", Joint Annual Meeting of WSSA and CWSS, Vancouver, BC, February 4, 2014 (Oral Presentation).  
	}%
\end{minipage}%

\section{SELECTED AWARDS}
\begin{minipage}{\maincolumnwidth}%
	\small{
	- Purdue Graduate Student Government Travel Award, Purdue University, 2015\\
          - Purdue Graduate Student Government Professional Award, Purdue University, 2014\\
          - Student Innovator Award, Purdue University, 2014\\
          - Summer Institutes in Statistical Genetics Travel Award, University of Washington, 2014\\
	- Best Oral Presentation, Department of Biological Sciences, Purdue University, 2014\\
	 - Dr. P.T. Gilham Graduate Award. Purdue University, 2011
	}%
\end{minipage}%
 
\section{TEACHING AND LEADERSHIP}
\begin{minipage}{\maincolumnwidth}%
	\small{
          - Assistant Program Coordinator, LSAMP Summer Undergraduate Research Program, 2016\\
          - President/Treasurer/Webmaster, Biological Graduate Student Council, Purdue University, 2012-15\\
          - Organizer/Lead Instructor, Computational Interdisciplinary Graduate Programs RNA-Seq Workshop, 2015\\
          - Webmaster, Society of Industrial and Applied Mathematics, Purdue University, 2012-13\\
	- Teaching Assistant, Purdue University\\
		\tab \quad  \bullet Introduction to R and Bioconductor, Summer 2014\\ 
		\tab \quad \bullet  Human Anatomy and Physiology Fall 2011 – Spring 2013
	}%
\end{minipage}%

}
% Publications from a BibTeX file without multibib
%  for numerical labels: \renewcommand{\bibliographyitemlabel}{\@biblabel{\arabic{enumiv}}}% CONSIDER MERGING WITH PREAMBLE PART
%  to redefine the heading string ("Publications"): \renewcommand{\refname}{Articles}
\nocite{*}
\bibliographystyle{plain}
\bibliography{publications}                        % 'publications' is the name of a BibTeX file

% Publications from a BibTeX file using the multibib package
%\section{Publications}
%\nocitebook{book1,book2}
%\bibliographystylebook{plain}
%\bibliographybook{publications}                   % 'publications' is the name of a BibTeX file
%\nocitemisc{misc1,misc2,misc3}
%\bibliographystylemisc{plain}
%\bibliographymisc{publications}                   % 'publications' is the name of a BibTeX file

%-----       letter       ---------------------------------------------------------

\end{document}


%% end of file `template.tex'.
